packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -0,0 +1,401 @@
+---
+title: Badge - Components
+description: A badge is a visually highlighted text label that describes an object.
```

Comment:
Are we going with this description or this one: `A badge is a small chunk of information that describes an object in a visually highlighted way.`?

I think I prefer this one a bit more because it emphasizes that Badge is a text label. what do you think?